### DISCUSSION TITLE
Upload file not working-MIME multipart message is not complete.

### DISCUSSION THREAD

**Discussion by priya-dsp** [artilleryio/artillery#3328] [Artillery]
Hi @hassy, we are trying to upload pdf file in post request. As per the artillery document, we tried all the listed ways but we are getting error: Unexpected end of MIME multipart stream. MIME multipart message is not complete. How to fix this ?
latest version of artillery is being used. 
we have post request where we are passing a parameter with json body and along with that we are uploading pdf file. 
- post:
            url: "/documents/images"
            headers:
                  Client: "{{ClientID}}"
                  Authorization: "Bearer {{AccessToken}}"
                  Content-Type: "multipart/form-data; boundary=6520f425-9fd5-4151-992e-996ad90ebed9"
                  Host: url of environment
                  Content-Length: 7435
                  Expect: 100-continue
            formData:
                  ImageDetails: '{json body}'
                  ImageBytes: 
                     fromFile: '\\sample.pdf'
                     contentType: 'application/octet-stream'

**Comment by hassy** [comment]
hi @priya-dsp 👋 I see a few extra headers you're passing in there like Content-Type, Host, and Content-Length. Is there a reason you're doing that? You don't need to specify any of those if you use Artillery's `fromFile` attribute with `formData`.
